src/components/NavBar.js

```diff
@@ -9,28 +9,37 @@ import SearchBarAutoComplete from './search-bar/SearchBarAutoComplete';
 import fullLogo from '../images/vbb-full-logo.png';
 // import miniLogo from '../images/vbb-picture-logo.png';
 
-const NavBar = ({ logout, authToken }) => {
+/**
+ * NavBar
+ * Connected functional component
+ * Displays Navigation Header Bar
+ *
+ * @param { Type History } history from react router dom
+ * @param {redux store} authToken authentication token from redux store
+ * @param {redux action} logOut action from logout import as a connected component
+ */
+const NavBar = ({ history, authToken, logOut }) => {
   const signInSignOut = !authToken ? (
-    <Link to="/signin" key="link-1">
-      <Button key="1" style={{ marginTop: '15px', color: '#549bea' }}>
+    <Link to="/signin" key="link-signin">
+      <Button key="0" style={{ marginTop: '15px', color: '#549bea' }}>
         Sign In
         <LoginOutlined />
       </Button>
     </Link>
   ) : (
-      <>
-        <SearchBarAutoComplete/>
-    <Link to="/" key="link-1">
-      <Button
-        key="1"
-        style={{ marginTop: '15px', color: '#549bea' }}
-        onClick={logout}
-      >
-        Sign Out
-        <LogoutOutlined />
-      </Button>
-    </Link>
-      </>
+      <React.Fragment key="group">
```

Comment:
This is what fixed the "Warning: Each child in a list should have a unique "key" prop" issue.